### ISSUE TITLE
Search returning MySQL error - Unknown Column 'display_name.t'

### ISSUE THREAD

**Issue by WRasada** [pods-framework/pods#6050]
The code below is returning this error when I perform a search. Plugin version 2.7.28 but when I revert the plugin to 2.6.11.1 the search works fine.

**Error**
`Database Error; SQL: SELECT DISTINCT t.*, wpum.* FROM `wp_users` AS `t` INNER JOIN `wp_posts` ON `wp_posts`.`post_author`=`t`.`id` INNER JOIN `wp_usermeta` AS wpum ON (wpum.meta_key="last_name" AND wpum.user_id=t.id) WHERE ( ( `wp_posts`.`post_type`='wc_user_membership' AND `wp_posts`.`post_status`='wcm-active' ) AND ( `t`.`user_status` = 0 ) ) AND ( `display_name`.`t` LIKE '%John%' ) ORDER BY meta_value ASC, `t`.`display_name`, `t`.`ID` LIMIT 0, 25; Response: Unknown column 'display_name.t' in 'where clause'`

**Code**
```
function occ_list_users_function($atts) {
    $params = ['limit' => 25,
        'select' => ['t.*', 'wpum.*'],
        'join' => [
            'INNER JOIN `wp_posts` ON `wp_posts`.`post_author`=`t`.`id`',
            'INNER JOIN `wp_usermeta` AS wpum ON (wpum.meta_key="last_name" AND wpum.user_id=t.id)'
        ],
        'where' => "`wp_posts`.`post_type`='wc_user_membership' AND `wp_posts`.`post_status`='wcm-active'",
        'orderby'   => 'meta_value', //or 'meta_value_num'
    ];
    // Get pods object
    $pods = pods('user', $params);
    // Search bar
    $output = $pods->filters(['label' => 'Search Members']);
    // Loop through the records
    if ($pods->total() > 0) {
        while ($pods->fetch()) {
            if (occ_has_active_membership($pods->field('id'))) {
                $output .= occ_get_user_profile($pods);
            }
        }
    }
    $output .= $pods->pagination(['type' => 'advanced']);
    return $output;
}
add_shortcode('view_users', 'occ_list_users_function');```

**Comment by unknownnf** [comment]
I've managed to replicate the bug, as a temporary workaround please set `$params['meta_fields'] = false`. Didn't manage to get the query to run otherwise.

**Comment by vitorpradof** [comment]
I've got the same error.

What can we do to solve this problem?

```
SELECT DISTINCT 't'.* FROM 'alp_posts' AS 't' 
WHERE ( ( 't'.'post_status' IN ( "publish" ) ) AND ( 't'.'post_type' = "convite" ) ) AND ('post_title'.'t' LIKE '%30245%' ) 
ORDER BY 't'.'menu_order', 't'.'post_title', 't'.'post_date' 
LIMIT 0, 20;
```

My problem is with the where tag, more specifically on the second AND.

I can't find the code that creates this query.

**Comment by sc0ttkclark** [comment]
I'll get a maintenance release out tomorrow that addresses this.

**Comment by sc0ttkclark** [comment]
Fixed via 6647828d549e8f5db18e3ef4a41b0ccf20813991
